### ISSUE TITLE
how to handle OBS plugins in nixpkgs?

### ISSUE THREAD

**Issue by colemickens** [NixOS/nixpkgs#61482]
## Issue description
I'd like to use `wlrobs` to be able to easily screen record/stream a wlroots-based desktop (such as Sway).

However, I'm not sure the best way to accommodate this in a Nix/NixOS world.

**Comment by colemickens** [comment]
ref: https://hg.sr.ht/~scoopta/wlrobs

**Comment by NilsIrl** [comment]
#70750 is probably what you want.

**Comment by NilsIrl** [comment]
As mentioned https://github.com/NixOS/nixpkgs/blob/a23a985594bf7c3eb543bdaf1e7a9c5b6ddd6099/pkgs/applications/video/obs-studio/wlrobs.nix#L5-L7

there is no wrapper yet, so plugins have to be installed somewhat manually.

**Comment by colemickens** [comment]
Hey thanks @NilsIrl! Since filing this issue, I began packaging wlrobs and I include usage instructions on the readme at [nixpkgs-wayland](https://github.com/colemickens/nixpkgs-wayland).

I'm going to go ahead and close this out, especially since I no longer use OBS and it looks like native support is imminent at which point I will have no reason for plugins even if I went back to OBS. Someone else may wish to work on or track a proper issue for an OBS Plugin system in NixOS.
